### DISCUSSION TITLE
Separate Inputs for Start and End Date for DateRangePicker

### DISCUSSION THREAD

**Discussion by arvind0598** [mantinedev/mantine#1331] [Feature requests]
I've been using the DateRangePicker for a bit, but most users get a bit confused while using this component. It would be easier if there were two inputs boxes - one for the start date and another for the end date.

Most websites online (trip providers like Airbnb and Google) use this format, so I'd expect users to be more familiar with this layout.

We can probably add this as a separate flag within the DateRangePicker?

References:
https://www.airbnb.co.uk
https://www.google.com/travel/flights/

**Comment by rtivital** [comment]
Use two separate DatePicker components instead of DateRangePicker

**Reply by arvind0598** [reply]
That is what we used initially, but that doesn't have the highlighted range that date picker offers. What would be great is if the two dates in the range showed up as two separate inputs, but everything else stayed the same.

That is the kind of behaviour I see most commonly online.
